src/Console/Report/FixReport/ReporterFactory.php

```diff
@@ -48,7 +56,7 @@ public function registerBuiltInReporters(): self
         }
 
         foreach ($builtInReporters as $reporterClass) {
-            $this->registerReporter(new $reporterClass());
+            $this->registerReporter(new $reporterClass($this->fixers));
```

Comment:
why this change?

Reply:
To be able to get description of a fixer here https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7089/files#diff-96b8f6fe35684550b46a932958362784f237a3fe0135d7aa8d71421868ac5999R63

Reply:
I think it's very big dependency for quite small usage.
I also wonder if we need whole FixerInterface functionality, or we only pass definitions (or even only descriptions)

Reply:
Without description it would look like this in Gitlab. Notice that Psalm has better description, because it exposes actual description instead of only rule name

<img width="536" alt="image" src="https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/assets/496233/265269bb-af3c-4f89-96ba-08cc7df47bda">

As for why Gitlab only: Because I'm not familiar with other reporters. Gitlab is something I use only at the moment for this. If somebody want to use it different formatter, after my change it will be straightforward now :)

Anyways I don't want to force this change, I am just trying to have a best result. If you say it can't be done then fine.

Reply:
@ostrolucky can you share how it looks like with the description, so we can compare it visually?

Reply:
This is an approximation, but I think it will look like this
<img width="779" alt="image" src="https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/assets/496233/b2e62b8c-520a-4d9a-a20f-a5161d6fcde3">

Reply:
Personally I prefer version **without** description. It makes much more sense to display error message if the tool reports it (like from Psalm or PHPStan), but fixer's description is misleading here - it's neither an error message, nor proper requirement. Some descriptions may be longer too, so considering multiple occurrences, it would make such output really messy. I believe it would be better with fixer's name only (if possible, with link to `https://cs.symfony.com/doc/rules/basic/<NAME>.html`, but only for built-in fixers).

Reply:
I agree. I will make it the first one then.